Content.Shared/Salvage/Fulton/SharedFultonSystem.cs

```diff
@@ -113,6 +113,14 @@ private void OnFultonInteract(EntityUid uid, FultonComponent component, AfterInt
         if (args.Target == null || args.Handled || !args.CanReach)
             return;
 
+        if (TryComp<ContainerManagerComponent>(uid: args.User, out var manager))
+        {
+            if (manager.ContainsEntity(args.Target.Value))
+            {
+                return;
+            }
+        }
+
```

Comment:
As above, also seems better to just block all containers.

Reply:
Entities in containers is probably better since we kind of want them to transport crates/bodybags on expeditions.

Reply:
> As above, also seems better to just block all containers.

dissallowing all containters would disallow lockers and crates from being fultoned